challenges/2021-05-18-sum-of-squares/solution_test.go

```diff
@@ -0,0 +1,24 @@
+package solution
+
+import (
+	"fmt"
+	"testing"
+)
+
+func TestSumOfSquares(t *testing.T) {
+	tables := []struct {
```

Comment:
Do we want to add a description for the tests and add more tests?

Reply:
I didn't add a description for the tests. If this is a dealbreaker I might be able to get it pushed later today.